crates/lsp/src/beancount_data.rs

```diff
@@ -127,18 +131,16 @@ impl BeancountData {
         let binding = content.clone().to_string();
         let matches = cursor_qry.matches(&query, tree.root_node(), binding.as_bytes());
         let mut tags: Vec<_> = matches
-            .into_iter()
             .flat_map(|m| {
-                m.captures
-                    .iter()
-                    .map(|capture| text_for_tree_sitter_node(content, &capture.node))
+                convert(m.captures).map(|capture| text_for_tree_sitter_node(content, &capture.node))
             })
+            .cloned()
```

Comment:
This seems to be required because the new tree-sitter version returns a streamed iterator.